kivy_ios/toolchain.py

```diff
@@ -1354,6 +1354,13 @@ def recipes(self):
                     print("{recipe.name:<12} {recipe.version:<8}".format(recipe=recipe))
 
     def clean(self):
+        def clean_cache(recipe, ctx):
+            """ Remove download artifacts for this build. """
+            recipe_inst = Recipe.get_recipe(recipe, ctx)
+            recipe_inst.ctx = ctx
+            if exists(recipe_inst.archive_fn):
+                shutil.rm(self.archive_fn)
```

Comment:
`shutil.rm` is not a valid function and it raises an `AttributeError` when called.
Instead, you can use `os.unlink()` or something similar.

Reply:
Yup. Thanks, corrected. Fixed previously but never pushed...:-(